public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -108,6 +148,13 @@ export const ReceiversTable: FC<Props> = ({ config, alertManagerName }) => {
                 {receiver.name} {receiver.provisioned && <ProvisioningBadge />}
               </td>
               <td>{receiver.types.join(', ')}</td>
+              {errorStateAvailable && (
+                <td>
+                  <ReceiverHealth
+                    errorsByReceiver={contactPointsState ? errorsByReceiver(contactPointsState, receiver.name) : 0}
```

Comment:
Is it possible for `contactPointsState` to be undefined?

Reply:
when we render this component for the first time and we still don't have any value in the Resux store